workflow/process.go

```diff
@@ -314,7 +314,8 @@ func (wf *workflowProcess) handleMessage(msg rrt.Message) error {
 		}
 
 	case *rrt.Panic:
-		return errors.E(op, errors.Str(command.Message))
+		// do not wrap error to pass it directly to Temporal
+		return bindings.ConvertFailureToError(msg.Failure, wf.env.GetDataConverter())
```

Comment:
```suggestion
		return errors.E(op, bindings.ConvertFailureToError(msg.Failure, wf.env.GetDataConverter()))
```

Reply:
In that case, the original failure information will be lost as temporal SDK won't be able to read the failure from the internals of the error.